docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "\n",
+    "from qiskit_experiments.test.t2hahn_backend import T2HahnBackend\n",
+    "# FakeJob is a wrapper for the backend, to give it the form of a job\n",
+    "from qiskit_experiments.test.utils import FakeJob\n",
+    "\n",
+    "\n",
+    "estimated_t2hahn2 = 20\n",
+    "# The behavior of the backend is determined by the following parameters\n",
+    "backend2 = T2HahnBackend(\n",
+    "    t2hahn=[20],\n",
+    "    frequency=[100100],\n",
+    "    initialization_error=[0.0],\n",
+    "    readout0to1=[0.02],\n",
+    "    readout1to0=[0.02],\n",
+    "    conversion_factor=conversion_factor,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "dt_factor2 = apply_prefix(1, unit2)\n",
+    "\n",
+    "# exp2.set_analysis_options(p0={\"amp\": 0.5, \"tau\": 20 * conversion_factor, \"base\": 0.5})\n",
+    "\n",
+    "expdata2 = exp2.run(backend=backend2, shots=2000)\n",
+    "expdata2.block_for_results()  # Wait for job/analysis to finish.\n",
+    "\n",
+    "# Display the figure\n",
+    "display(expdata2.figure(0))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###  $T_{2}$ v.s  $T_{2}^{\\ast}$\n",
```

Comment:
```suggestion
    "###  $T_{2}$ versus  $T_{2}^{\\ast}$\n",
```
v.s. (with a dot after the s) is also fine

Reply:
Changed as suggested.